### DISCUSSION TITLE
What is Overlays plugin main use case?

### DISCUSSION THREAD

**Discussion by juanbrk** [mistic100/Photo-Sphere-Viewer#1154] [General]
I'm working on a project where we display markers on top of a panorama that the user can interact with (mainly hover and display a tooltip, edit or delete them in-situ) and there are some other, completely different, markers whose sole purpose is to click on them and load a different panorama. Since the introduction of [Overlays plugin](https://photo-sphere-viewer.js.org/plugins/overlays.html#usage) I have been tasked with replacing the latter markers with overlays.  This is due to these markers being only clickable, same as overlays (as seen on the [documentation](https://photo-sphere-viewer.js.org/plugins/overlays.html#example)), and because we want to separate concerns and responsibilities: 
- Interactive markers on one side
- Non-interactive markers as overlays.

So, my question here is: What is the main purpose of overlays, and how do these differ from Markers? If there were a Venn diagram with Markers and Overlays use cases on each side, would there be a use-case intersection of both? 
I've already read all the documentation and sense that we should stick with Markers on both scenarios, but I'm not convinced yet. Also, just to point this out, these markers are generated programmatically and there would be more than one on the panorama image at the same time.

**Comment by mistic100** [comment]
Given your description you should absolutely stick to markers.

As the documentation says overlays are for **big** elements, covering the whole panorama (or a good part of it), this was originally a [core feature](https://photo-sphere-viewer.js.org/guide/config.html#overlay) (and still is a version 5.4). You cannot achieve the "x-ray" effect of the overlays demo with a marker, the plugin allowing smaller overlays is just a bonus.

It was also implemented as a personal (non working) prototype to reimplement [Myst 3 game](https://youtu.be/nrZabBlIY0o?t=97) which features cubemap scenes with embedded videos.
